### DISCUSSION TITLE
how do i use variables in an href?

### DISCUSSION THREAD

**Discussion by normrubin** [quarto-dev/quarto-cli#9849] [Q&A]
### Description

 Im trying to use variables in  _quarto.yml   it looks like variables in an href do no expand?  Am I using the wrong syntax?
 

my _variables.yml file ------------

bril:   bril_expand

my _quarto.yml file 

project:
  type: website
  title: "{{< var bril >}}"

website:
  sidebar:
    contents:
      - text: "Bril Documentation"
        href: "{{< var bril >}}"

my index.qmd file 
---
title: "junk"
---

index 
Version {{< var bril >}} is a minor upgrade.

I'm  use the bril variable in 3 places 
in index.qmd  works 
in _quarto.yml title works
in _quarto.yml href - does not expand

**Comment by normrubin** [comment]
sorry for the crap format - This is my first post so I don't know how to put in a repro case

**Reply by mcanouil** [reply]
Could you properly format your post using code blocks for code and terminal outputs? Thanks.
If your code contains code blocks, you need to enclose it using more backticks, *i.e.*, usually four ` ```` `.

Note that the formatting guidelines were provided when you opened the discussion.

---

You can share a **self-contained "working" (reproducible)** Quarto document using the following syntax, _i.e._, using more backticks than you have in your document (usually four ` ```` `).
See <https://quarto.org/bug-reports.html#small-is-beautiful-aim-for-a-single-document-with-10-lines>.

If you have multiple files (and if it is **absolutely required** to have multiple files), please share as a Git repository.

<table>
<tr><th>R</th><th>Python</th></tr>
<tr><td>

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
engine: knitr
---

This is a reproducible Quarto document.

```{r}
x <- c(1, 2, 3, 4, 5)
y <- c(1, 4, 9, 16, 25)

plot(x, y)
```

![An image](https://placehold.co/600x400.png)

The end.
````
`````

</td><td>

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
engine: jupyter
---

This is a reproducible Quarto document.

```{python}
import matplotlib.pyplot as plt

x = [1, 2, 3, 4, 5]
y = [1, 4, 9, 16, 25]

plt.plot(x, y)
plt.show()
```

![An image](https://placehold.co/600x400.png)

The end.
````
`````

</td></tr>
</table>

Additionally and if not already given, please share the output of `quarto check` within a code blocks (*i.e.*, using three backticks ` ```txt `), see <https://quarto.org/bug-reports.html#check>.
